drivers/SmartThings/zigbee-switch/profiles/switch-power-energy-consumption-report-aqara-t1.yml

```diff
@@ -0,0 +1,29 @@
+name: switch-power-energy-consumption-report-aqara-t1
+components:
+  - id: main
+    capabilities:
+      - id: switch
+        version: 1
+      - id: powerMeter
+        version: 1
+      - id: energyMeter
+        version: 1
+      - id: powerConsumptionReport
+        version: 1
+      - id: firmwareUpdate
+        version: 1
+      - id: refresh
+        version: 1
+    categories:
+      - name: SmartPlug
+preferences:
+  - preferenceId: stse.maxPowerCN
```

Comment:
FYI.

Different regions have different limits on maximum power.
So I created 2 new preferences stse.maxPowerCN and stse.maxPowerUS  as an extension to "stse.maxPower".

stse.maxPower (EU/KR): 100W ~ 2300W
stse.maxPowerCN (CN): 100W ~ 2500W
stse.maxPowerUS (US):  100W ~ 1800W

Reply:
So what does this preference do? I'm afraid I don't understand.

Reply:
This function is to set the power limit of the electrical equipment, once the set power is exceeded, the socket will start the protection mechanism. Automatic power off to protect the safety of the user's home.

Reply:
Shouldn't the labels used be more descriptive, then? Will the user know what "1" sets the power limit to? Or is that handled in the custom presentation?

Reply:
The user chooses power on the ST app instead of 1, 2, 3, 4, 5, etc. This is a screenshot of the ST app Settings.
![1](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/assets/150672162/d4d7425c-588f-48df-a92c-3b0eb6a1cace)
![2](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/assets/150672162/a92c32db-7f39-40a0-9fc9-d23281cdc919)